### PR TITLE
Fixes 1284: Improve test cases for URL unshortener

### PR DESCRIPTION
**Pull request by vibhcool**
This PR fixes Issue #1284

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by vibhcool** [comment]
tested and open for review 
@mariobehling @singhpratyush @kavithaenair @hemantjadon @Achint08 @djmgit @SarishiNoHara @SKrPl

File: test/org/loklak/harvester/RedirectUnshortenerTest.java [hunk 1]
```diff
@@ -13,17 +13,18 @@
...
+        shortlinkMap.put("http://tinyurl.com/loklak-server", "https://github.com/loklak/loklak_server/wiki");
+        shortlinkMap.put("https://t.co/raRRie3ado", "https://medium.com/javascript-scene/angular-2-vs-react-the-ultimate-dance-off-60e7dfbc379c");
+        shortlinkMap.put("http://bit.ly/2tvkGa1", "https://medium.com/@keekri17/my-adventures-at-foss-asia-d1a5d462b792");
+        shortlinkMap.put("http://ow.ly/P9SX30ddSTI", "https://github.com/loklak/loklak_server/issues/1284");
+        shortlinkMap.put("http://bit.do/blog-fossasia", "http://blog.fossasia.org/");
         shortlinkMap.put("http://fb.me/4lcXZsyyO", "https://www.facebook.com/permalink.php?story_fbid=1550262581900846&id=1381813572079082");
-        shortlinkMap.put("http://wp.me/p4yQu6-za0", "https://www.oklahomafoodandmusic.com/?p=135160");
+        shortlinkMap.put("http://wp.me/sf2B5-shorten", "https://en.blog.wordpress.com/2009/08/14/shorten/");
+        shortlinkMap.put("https://is.gd/gyk3VT", "https://github.com/fossasia/");
+        //Double shortened link
```
Comment (singhpratyush) [inline test/org/loklak/harvester/RedirectUnshortenerTest.java]: BTW there are more links that are shortened more than once in the list (and they were earlier there too).

File: test/org/loklak/harvester/RedirectUnshortenerTest.java [hunk 2]
```diff
@@ -13,17 +13,17 @@
     @Test
     public void testRedirectUnshortener() {
         HashMap<String, String> shortlinkMap = new HashMap<>();
-        shortlinkMap.put("http://tmblr.co/Z6YPNx1jL1hHK", "http://myjourneymyway.tumblr.com/post/117390619732/first-time-making-loklak-my-grandma-would-be#_=_");
-        shortlinkMap.put("http://dlvr.it/8kTDbJ", "http://islamtimes.org/en/doc/news/442892/?utm_source=dlvr.it&utm_medium=twitter");
```
Comment (daminisatya) [inline test/org/loklak/harvester/RedirectUnshortenerTest.java]: I find these URLs working. can you explain why this change?

Comment (vibhcool) [inline test/org/loklak/harvester/RedirectUnshortenerTest.java]: there have been cases that this test failed because of small changes in the websites' url like `http` to `https` . So I and @singhpratyush got urls of some websites(which we assumed) will not make changes in their urls too often. This will let this test fail less often and more reliable. :)
